config/locales/hyrax.zh.yml

```diff
@@ -8,6 +8,7 @@ zh:
       messages:
         conflict: 您的更改无法保存，因为另一用户(或后台作业）在您开始编辑后更新了%{model}。 请确保所有文件附件已成功完成之后再试一次。这已经是最新刷新后保存的副本%{model}
   blacklight:
+    application_name: must be here for features page to open
```

Comment:
Ah, good call! So in Hyku, we did this: https://github.com/samvera-labs/hyku/commit/e9430e27cd82343fcf7ed08d051b7a78629dce35

I'd ask the Blacklight folks for advice on how we should be handling this. Should we find someone to add translations directly to Blacklight, or in the meantime should Hyrax ship some Blacklight translations? @jkeck @mejackreed @cbeer @jcoyne